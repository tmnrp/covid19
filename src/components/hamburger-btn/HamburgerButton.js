import "./HamburgerButton.scss";
import React from "react";
import { FaBars } from "react-icons/fa";

const HamburgerButton = () => {
  return (
    <div className="hamburger-btn primary-color">
      <FaBars />
    </div>
  );
};

export default HamburgerButton;
