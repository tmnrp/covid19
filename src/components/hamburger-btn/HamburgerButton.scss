@import "../../index.scss";

.hamburger-btn {
  display: flex;
  font-size: 25px;
  padding: 5px;
  cursor: pointer;
}
