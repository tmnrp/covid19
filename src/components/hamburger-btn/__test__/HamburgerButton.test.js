import React from "react";
import ReactDOM from "react-dom";
import HamburgerButton from "../HamburgerButton";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("Renders with out crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<HamburgerButton />, div);
});

it("Snapshot the hamburger menu", () => {
  const tree = renderer.create(<HamburgerButton />);
  expect(tree).toMatchSnapshot();
});
