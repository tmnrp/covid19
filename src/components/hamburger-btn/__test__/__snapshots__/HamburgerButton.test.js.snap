// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot the hamburger menu 1`] = `
<div
  className="hamburger-btn primary-color primary-color-darken-hover"
  onClick={[Function]}
>
  <svg
    fill="currentColor"
    height="1em"
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 448 512"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
    />
  </svg>
</div>
`;
